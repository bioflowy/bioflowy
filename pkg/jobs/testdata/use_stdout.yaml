children:
  - args:
      - "echo"
      - "this stdout test"
    stdout:
      pipe: "pipe"
  - args:
      - "sh"
      - "-c"
      - "nl $(pipe) > use_stdout.yaml.result"
    inputs:
      - "pipe"
