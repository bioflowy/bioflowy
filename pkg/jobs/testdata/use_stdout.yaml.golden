     1	this stdout test
